sdks/python/apache_beam/yaml/examples/transforms/elementwise/explode.yaml

```diff
@@ -0,0 +1,31 @@
+# coding=utf-8
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+pipeline:
+  type: chain
+  transforms:
+    - type: Create
+      name: Gardening plants
+      config:
+        elements:
+          - produce: ['🍓Strawberry', '🥕Carrot', '🍆Eggplant', '🍅Tomato', '🥔Potato']
```

Comment:
Not sure how clear this example is. Maybe add a non-exploding field?

Reply:
Done.